client/vscode/src/core/idealsClient.ts

```diff
@@ -89,16 +91,38 @@ export class IdealsClient {
       };
     }
 
-    let ideaExecutablePath = process.env.IDEALS_IJ_PATH;
+    let ideaExecutablePath: string | undefined =
+      vscode.workspace.getConfiguration('ideals').get('startup.ideaExecutablePath') || process.env.IDEALS_IJ_PATH;
 
     if (!ideaExecutablePath) {
-      throw new Error("Path to IntelliJ IDEA executable must be specified in environment variable IDEALS_IJ_PATH");
+      throw new Error("Path to IntelliJ IDEA executable must be specified in extension configuration or in environment variable IDEALS_IJ_PATH");
+    }
+
+    const ideaVersionDir = path.normalize(path.dirname(path.dirname(ideaExecutablePath)));
+    const ideaVersion = path.basename(ideaVersionDir);
+    const dirWithVmOptions = path.dirname(ideaVersionDir);
+    let vmoptionsPath = path.join(dirWithVmOptions, path.basename(ideaVersion) + ".vmoptions");
```

Comment:
I'm not sure this will work as expected in case of path to a script is provided instead of binary executable